python_modules/dagster/dagster/_core/telemetry.py

```diff
@@ -538,6 +545,49 @@ def get_stats_from_external_repo(external_repo: "ExternalRepository") -> Mapping
     }
 
 
+def get_resource_stats(external_resources: Sequence["ExternalResource"]) -> Mapping[str, str]:
+    dagster_resources = [
+        "dagster_snowflake.resources.SnowflakeResource",
+        "dagster_dbt.cli.resources.DbtCliClientResource",
+        "dagster_snowflake.resources.snowflake_resource"
+        # fill out with rest of resources
+    ]
+
+    dagster_io_managers = [
+        "dagster_snowflake_pandas.snowflake_pandas_type_handler.SnowflakePandasIOManager",
```

Comment:
would it be possible to separate module name and class name, so something like:
```
{"module_name": "dagster_snowflake_pandas", "class_name": "dagster_snowflake_pandas.snowflake_pandas_type_handler.SnowflakePandasIOManager"}
```
so we can easily tell whether it's a custom or builtin resource by the module name in post processing